data/coalition/coalition outfits.txt

```diff
@@ -249,6 +249,29 @@ outfit "Large Steering Module"
 	"turning heat" 8.2
 	description "By focusing on small, modular engines rather than the massive capital ship drives that most other species develop, the Coalition loses some of the potential efficiency for very large ships, but gains a considerable amount of flexibility."
 
+outfit "Refuelling Module"
+	category "Systems"
+	cost 822000
+	thumbnail "outfit/refuelling module"
+	"mass" 9
+	"outfit space" -9
+	"fuel capacity" -100
+	"fuel generation" .05
+	"energy consumption" .025
+	description "To avoid the risk of being inspected by the Heliarch as they refuel on planets, Resistance ships employ this outfit when carrying supplies or stolen outfits."
+	description "	The outfit's mechanisms require a set amount of fuel to be put aside for its inner workings, so a ship can only install so many of these before becoming incapable of jumping to another system."
+
+outfit "Decoy Plating"
+	category "Systems"
+	cost 93000
+	thumbnail "outfit/decoy plating"
+	"mass" 7
+	"outfit space" -7
+	"scan interference" 1.2
+	"radar jamming" 4
+	description "This plating takes advantage of a ship's scan logs, reproducing the data of other ships close by and effectively deflecting attempts to scan a ship's true cargo and equipment."
```

Comment:
```suggestion
	description "This special plating is able to take advantage of a ship's scan logs. By mimicking the scan data of other ships close by, this plating is able to effectively hide a ship's true cargo and equipment from scanning."
```

Reply:
didn't want "is able to" to be repeating so didn't do that on the second instance